# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/kmod/hotfix-cloexec.patch.avr32
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- kmod-33/libkmod/libkmod-file-zlib.c.vanilla	2024-12-14 23:02:25.305431232 +0100
+++ kmod-33/libkmod/libkmod-file-zlib.c	2024-12-14 23:02:50.885431181 +0100
@@ -29,9 +29,11 @@
 	int gzfd;
 
 	errno = 0;
+# if 0
 	gzfd = fcntl(file->fd, F_DUPFD_CLOEXEC, 3);
 	if (gzfd < 0)
 		return -errno;
+# endif
 
 	gzf = gzdopen(gzfd, "rb"); /* takes ownership of the fd */
 	if (gzf == NULL) {
